public/video-ui/src/components/FormFields/TextInputTagPicker.js

```diff
@@ -5,7 +5,6 @@ import UserActions from '../../constants/UserActions';
 import TagTypes from '../../constants/TagTypes';
 import CapiSearch from '../CapiSearch/CapiSearch';
 import removeStringTagDuplicates from '../../util/removeStringTagDuplicates';
-import removeTagDuplicates from '../../util/removeTagDuplicates';
```

Comment:
oh, intellij removed this as it wasn't being used... sounds similar to what we're now doing below 🤷‍♂️